packages/project-disaster-trail/src/components/Game/Orb.js

```diff
@@ -1,6 +1,7 @@
 /** @jsx jsx */
-import { Component } from "react";
+import { PureComponent } from "react";
 import { jsx, css } from "@emotion/core";
+import { returnStatement } from "@babel/types";
```

Comment:
What's this do?

Reply:
I will remove that. Sometimes my auto-imports give me whack results.

Reply:
Removed!